install/Create_System_Configuration_Using_Cluster_Discovery_Service.md

```diff
@@ -180,15 +180,22 @@ The following steps verify the status and lists the IP addresses of nodes, fabri
 1. (`pit#`) Verify the status and list the IP addresses of the nodes.
 
    ```bash
-   "${CVT_PATH}"/cds/discover_status.py nodes --out
+   "${CVT_PATH}"/cds/discover_status.py nodes -o
    ```
 
+   > **NOTE**:
+   >
+   > - Ensure that all the NCNs and CNs are discovered and listed. If any nodes are not discovered, identify the nodes using the MAC address and verify the switch connectivity of those nodes.
+   > - If the discover status step is rerun after a long duration, make sure that to start the procedure from the beginning of [Configure the management network](#2-configure-the-management-network) section.
```

Comment:
Update:
If the discover status step is rerun after a long duration, make sure that to start the procedure from the beginning of [Configure the management network](#2-configure-the-management-network) section.

To:
If the discover status step is rerun after a long duration, start the procedure from the beginning of section [Configure the management network](#2-configure-the-management-network).